.storybook/preview.js

```diff
@@ -73,3 +73,4 @@ export const parameters = {
     ],
   },
 };
+export const tags = ['autodocs'];
```

Comment:
Can we know what this is supposed to help with?

Reply:
Generates autodocs for components that have been tagged 'autodocs'